android-example/build.rs

```diff
@@ -159,22 +204,44 @@ where
     Ok(())
 }
 
-fn rust_swig_expand(from: &Path, out: &Path) -> Result<(), String> {
+fn rust_swig_expand(source_dir: &Path, file: &Path, out_dir: &Path) -> Result<(), String> {
+    let app_id = env::var("ANDROID_APPLICATION_ID")
```

Comment:
At now `cargo check --target=arm-linux-androideabi` failed for this project.
I use Android Studio for Rust development via Rust plugin, and run `cargo check`
for fast syntax check via "Cargo Command" option (see https://users.rust-lang.org/t/intellij-rust-and-compile-errors/15103), it is impossible to set `ANDROID_APPLICATION_ID` environment variable per project.
So may be use some other way, like get it from file, may be from `settings.gradle` ?

Reply:
Agree